### PR TITLE
Editor can now load any old plio using new REST APIs

### PR DESCRIPTION
**Pull request by deepansh96**
## Summary
- [x] Refactored `getPlio()` function in plio API services to fetch the specified plio, all of it's items (along with the questions), and prepare it into an object that can be directly consumed by the `Editor.vue`

## Test Plan
Tested locally by creating dummy plios/items/questions

### CODE REVIEW

File: src/pages/Editor.vue [hunk 1]
```diff
@@ -797,10 +796,10 @@ export default {
...
       // the question is coming from.
       // currently the source is only "default" as questions will be created on the editor only
-      var metadata = {};
-      metadata["source"] = {};
-      metadata["source"]["name"] = "default";
-      return metadata;
+      var meta = {};
+      meta["source"] = {};
+      meta["source"]["name"] = "default";
+      return meta;
```
Comment (deepansh96) [inline src/pages/Editor.vue:802]: changed to `meta` because that is the name of the key we're keeping in the DB

File: src/services/API/Plio.js [hunk 2]
```diff
@@ -29,15 +43,9 @@ export default {
   createPlio() {
     // creates a new draft plio
     var newPlioData = {
-      value: {
-        status: "draft",
-      },
-      bucket: "plio-test-data",
+      created_by: 3,
```
Comment (deepansh96) [inline src/services/API/Plio.js:46]: hardcoded for now
When auth is completed, the user id being passed here will be parametrised

File: src/pages/Editor.vue [hunk 3]
```diff
@@ -620,10 +620,9 @@ export default {
     },
     async loadPlio() {
       // fetch plio details
-      PlioService.getPlio(this.plioId).then((response) => {
-        var plioDetails = response.data.plioDetails;
+      await PlioService.getPlio(this.plioId).then((plioDetails) => {
         this.items = plioDetails.items || [];
-        this.videoId = plioDetails.video_id || "";
+        this.videoId = "";
```
Comment (dalmia) [inline src/pages/Editor.vue:625]: not needed right?
```suggestion
```

File: src/services/API/Plio.js (hunk 2 above)
Comment (dalmia) [reply src/services/API/Plio.js:46]: change it to 1

**Review by dalmia — CHANGES_REQUESTED** [review]
couple minor changes

**Comment by dalmia** [comment]
tested locally - works!

File: src/services/API/Plio.js [hunk 4]
```diff
@@ -32,6 +32,7 @@ export default {
       plioDetails.plioTitle = plio.data.name;
       plioDetails.status = plio.data.status;
       plioDetails.updated_at = plio.data.updated_at;
+      plioDetails.plioDbId = plio.data.id;
```
Comment (dalmia) [inline src/services/API/Plio.js:35]:
```suggestion
      plioDetails.plioDBId = plio.data.id;
```

**Review by dalmia — CHANGES_REQUESTED** [review]
1 minor change - I'll do it.

**Review by dalmia — APPROVED** [review]

**Pull request merged**